### PR TITLE
initial react conversion. still needs a lot of work to be more modular

### PR DESCRIPTION
**Pull request by davidmenendez**

### CODE REVIEW

**Comment by prestonp** [comment]
this is awesome

**Comment by davidmenendez** [comment]
@prestonp we need you bro!

File: src/components/app.jsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    this.setMute = this.setMute.bind(this);
+    this.setVoice = this.setVoice.bind(this);
+
+    socket.emit('joinroom', room);
+
+    socket.on('reconnect', () => {
+      socket.emit('joinroom', room);
+    });
+
+    socket.on('notification', (msg) => {
```
Comment (prestonp) [inline src/components/app.jsx]: Could shorten to `socket.on('notification', Notify);`

File: src/components/soundboard.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import Button from './button.jsx';
+
+class Soundboard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const sounds = this.props.sounds.filter((sound) => {
```
Comment (prestonp) [inline src/components/soundboard.jsx]: Nice! Sound filter is 🔥

**Pull request merged**